CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 ## [Unreleased]
 
 ### Added
+
 - [x2gbfs 2024-12-19: add Cantamen provider teilauto_schwaebisch_hall](https://github.com/mobidata-bw/x2gbfs/blob/main/CHANGELOG.md#2024-12-19)
+- [OCPDB 1.3: Mitigate OCPI compatibility issues](https://github.com/binary-butterfly/ocpdb/blob/cc53f403c32e8d9a645064b36f22b9b3992060f5/CHANGELOG.md#version-130)
+
+⚠️ The OCPDB changes might have effects on clients, as it changes the type if `id` from integer (which was wrong) to
```

Comment:
You could clarify where this type gets changed? Just in the internal database? In the JSON response bodies?

The [linked Issue](https://github.com/binary-butterfly/ocpdb/issues/81) contains a lot of discussion, so having a summary here would help.

Reply:
Done